schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/stac-to-iso19115-3.xsl

```diff
@@ -139,6 +141,21 @@
                                         else 'STAC Collection'" />
                 </gco:CharacterString>
               </cit:title>
+              <xsl:if test="sci_doi">
+                <cit:identifier>
+                  <mcc:MD_Identifier>
+                    <mcc:code>
+                      <gcx:Anchor xlink:href="{concat('https://doi.org/', sci_doi)}"><xsl:value-of select="sci_doi"/></gcx:Anchor>
```

Comment:
The extension that this field comes from (https://github.com/stac-extensions/scientific) says that a `cite-as` link should be present in the collection as well and give us a link to the actual DOI, without having to assemble this link ourselves. But in the sample used for the test this link doesn't exist, so I guess your approach is safer.